src/intersection-observer-polyfill.js

```diff
@@ -411,7 +413,9 @@ function calculateChangeEntry(
   }
 
   return /** @type {!IntersectionObserverEntry} */ ({
-    time: performance.now(),
+    time: (typeof performance !== 'undefined' &&
+        typeof performance.now !== 'undefined') ? performance.now()
```

Comment:
`(performance && performance.now) ? performance.now() : ...`. The `performance.now` check is probably unnecessary anyways.